src/daemon/ubuntu_image_host.cpp

```diff
@@ -280,12 +295,11 @@ mp::SimpleStreamsManifest* mp::UbuntuVMImageHost::manifest_from(const std::strin
 {
     check_remote_is_supported(remote);
 
-    update_manifests();
-
-    auto it = std::find_if(manifests.begin(), manifests.end(),
-                           [&remote](const std::pair<std::string, std::unique_ptr<SimpleStreamsManifest>>& element) {
-                               return element.first == remote;
-                           });
+    const auto it =
+        std::find_if(manifests.cbegin(), manifests.cend(),
+                     [&remote](const std::pair<std::string, std::unique_ptr<SimpleStreamsManifest>>& element) {
+                         return element.first == remote;
+                     });
```

Comment:
Another `clang-format` only change...

Reply:
This one is fine, I prepended const (an actual code change) which makes different line breaks. The other one probably is.

Reply:
Ah, I see, a subtle change.  Thanks!